src/app/pages/HomePage/Graph/GraphTooltip/index.tsx

```diff
@@ -1,18 +1,18 @@
 import { styled, useTheme } from '@mui/material/styles'
 import Tooltip, { TooltipProps, tooltipClasses } from '@mui/material/Tooltip'
-import { FC } from 'react'
+import { FC, ReactNode } from 'react'
 import Box from '@mui/material/Box'
 import { COLORS } from '../../../../../styles/theme/colors'
 import AccessTimeIcon from '@mui/icons-material/AccessTime'
 import Typography from '@mui/material/Typography'
-import AdjustIcon from '@mui/icons-material/Adjust'
 import CheckCircleIcon from '@mui/icons-material/CheckCircle'
 import useMediaQuery from '@mui/material/useMediaQuery'
 import { useTranslation } from 'react-i18next'
 import { TFunction } from 'i18next'
 import { RouteUtils } from '../../../../utils/route-utils'
 import { useNavigate } from 'react-router-dom'
 import { Layer } from '../../../../../oasis-indexer/api'
+import blockchainWhiteImage from '../images/blockchain-white.svg'
```

Comment:
We should probably consolidate the icon with the one in the `src/app/components/PageLayout/images/blockchain.svg`

Reply:
good idea, we can do it in next PRs